extensions/wikia/PortableInfoboxBuilder/js/PortableInfoboxBuilderPonto.js

```diff
@@ -19,7 +19,7 @@ define('wikia.infoboxBuilder.ponto', ['wikia.window', 'ponto'], function (window
 		 * @returns {Boolean}
 		 */
 		this.redirectToTemplatePage = function (title) {
-			window.location = window.location.origin + '/wiki/Template:' + title;
+			w.location = w.location.origin + '/wiki/Template:' + title;
```

Comment:
how this gonna work for templates like: http://marvel.wikia.com/wiki/Marvel_Database:Comic_Template ?